src/MongoDB.Driver.GridFS/GridFSBucketOptions.cs

```diff
@@ -99,6 +102,18 @@ public int ChunkSizeBytes
                 _chunkSizeBytes = value;
             }
         }
+        
+        /// <summary>
+        /// Gets or set whether or not GridFS will compute MD5 checksums of uploaded files.
```

Comment:
While true in a general sense this sounds backwards. How about:

Gets or sets whether to disable MD5 checksum computation when uploading a GridFS file.

Reply:
Ah, true. I'll use your summary.